pkg/k8s/apis/cilium.io/v2/types.go

```diff
@@ -292,6 +292,7 @@ type CiliumEndpointList struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:resource:singular="ciliumnode",path="ciliumnodes",scope="Cluster",shortName={cn,ciliumn}
 // +kubebuilder:storageversion
+// +kubebuilder:subresource:status
```

Comment:
I believe that you already scanned once to make sure this is only resource missing `subresource:status` :)